openshift/templates/backup/backup-deploy.prod.param

```diff
@@ -0,0 +1,38 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/backup/backup-deploy.json
+#=========================================================
+# NAME=backup
+# SUFFIX=
+# APP_NAME=Backup
+# APP_GROUP=sheriff-scheduling
+# SOURCE_IMAGE_NAME=backup
+# IMAGE_NAMESPACE=jag-shuber-tools
+TAG_NAME=prod
+# POSTGRESQL_SERVICE_NAME=postgresql
+# POSTGRESQL_USER_KEY_NAME=database-user
+# POSTGRESQL_PASSWORD_KEY_NAME=database-password
+# BACKUP_STRATEGY=rolling
+# WEBHOOK_URL=
+ENVIRONMENT_FRIENDLY_NAME=Sheriff Scheduling (prod)
+ENVIRONMENT_NAME=jag-shuber-prod
+# BACKUP_DIR=/backups/
+# DAILY_BACKUPS=
+# WEEKLY_BACKUPS=
+# MONTHLY_BACKUPS=
+# DATABASE_SERVER_TIMEOUT=120
+# CONFIG_FILE_NAME=backup.conf
+# CONFIG_MAP_NAME=backup-conf
+# CONFIG_MOUNT_PATH=/
+BACKUP_VOLUME_NAME=bk-jag-shuber-prod-kcupbl9x9rx0
```

Comment:
Is this reusing a pre-existing volume? I just noticed that bk-jag-shuber-prod-kcupbl9x9rx0 was created 4 months ago

Reply:
Yes, it's reusing (sharing) the existing volume.